tests/run.sh

```diff
@@ -204,6 +204,12 @@ for f in $tests; do
     # Send the test file to awesome.
     cat $f | DISPLAY=$D "$AWESOME_CLIENT" 2>&1
 
+    # Kill awesome after 1 minute (e.g. with errors during test setup).
+    (sleep 60; if kill -0 $awesome_pid 2>/dev/null; then
+        echo "Killing (stale?!) awesome (PID $awesome_pid) after 10 seconds."
+        kill $awesome_pid
+      fi) &
+
```

Comment:
@psychon 
I will add a check using `ps -o comm= $awesome_pid` here to make sure that it's awesome.

Does that sound good to you?